scripts/build-autocomplete.php

```diff
@@ -0,0 +1,14 @@
+<?php
+// are dev requirements installed?
+if ( file_exists('vendor/bin/symfony-autocomplete') ) {
+    $autocomplete = shell_exec('vendor/bin/symfony-autocomplete bin/terminus');
+    // d($autocomplete);
+    file_put_contents('assets/autocomplete.txt', $autocomplete);
+} elseif ( strlen(shell_exec('which symfony-autocomplete')) > 0 ) {
+    // global install?
+    $autocomplete = shell_exec('symfony-autocomplete bin/terminus');
+    file_put_contents('assets/autocomplete.txt', $autocomplete);
+} else {
+    echo "Please install dev dependencies, or run composer global require bamarni/symfony-console-autocomplete";
```

Comment:
The project in question recommends `composer global require`, but we should not.

https://pantheon.io/blog/fixing-composer-global-command

Reply:
Oh, never mind. bamarni/symfony-console-autocomplete has no dependencies, so it's actually safe to use it with `composer global require`.

Reply:
yeah, my assumption is that almost no one will ever run this, most people will get this via the CI generated Phar file. This will only be needed using Terminus via composer